chapter_preface/index_vn.md

```diff
@@ -242,7 +267,16 @@ at the expense of requiring the reader
 to trust us with some curatorial decisions.
 -->
 
-*dịch đoạn phía trên*
+Chúng tôi sẽ xen kẽ mã nguồn có thể thực thi với kiến thức nền tảng
+khi cần thiết.
+Thông thường, chúng tôi sẽ thường xuyên *err* ở phía tạo ra
+những công cụ có sẵn trước khi giải thích chúng đầy đủ (và chúng tôi sẽ
+theo sát bằng cách giải thích phần nền tảng sau).
+Ví dụ, chúng tôi có thể sử dụng *hạ gradient ngẫu nhiên* trước khi
```

Comment:
nên kèm thêm từ "thuật toán" sẽ dễ hiểu hơn.

sử dụng thuật toán "giảm gradient ngẫu nhiên"